samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,130 @@
+
+Testing
+=======
+
+The AT Client sample can be tested by running the following commands from the LTE Link Monitor terminal:
+
+#. Enter the command :command:`AT+CFUN?` 
+
+   * This command connects to an LTE network and triggers the command :command:`AT+CFUN=1` which will change the modem state to the normal mode.
+   * Enter the command :command:`AT+CFUN?` into the LTE Link Monitor terminal again. The UART/Modem/UICC/LTE/PDN indicators in the LTE Link Monitor side panel turns into green.
+   * This command also automatically launches a series of commands like:
+
+     * :command:`AT+CGSN=1` which displays the product serial idenitification number(IMEI). 
+     * :command:`AT+CGMI`   which displays the manufacturer name.
+     * :command:`AT+CGMM`   which displays the model identification name.
+     * :command:`AT+CGMR`   which displays the revision identification.
+     * :command:`AT+CEMODE` which displays the current mode of operation. 
+
+#. Enter the command :command:`AT+CLAC` 
+
+   This command displays a list of all the available AT commands.
+
+#. Enter the command :command:`AT%XOPERID`
+
+   This command returns the network operator ID.
+
+#. Enter the command :command:`AT%XMONITOR`
+
+   This command returns the modem parameters.
+
+#. Enter the command :command:`AT%XTEMP?`
+
+   This command results in the display of current modem temperature.
+
+
+
+
+Sample output
+=============
+
+The following image is a sample output of the command :command:`AT+CLAC`
+
+.. figure:: /images/ATCLAC.png
+   :height: 300
+   :width: 800
+   :alt: AT Client Sample Output for AT+CLAC
+
+The following image is a sample output of the command :command:`AT%XMONITOR`
+
+.. figure:: /images/AT%XMONITOR.png
+   :height: 100
+   :width: 800
+   :alt: AT Client Sample Output for AT%XMONITOR
+
+Dependencies
+************
+
+This sample uses the following libraries:
+
+From |NCS|
+
+* :ref:`at_cmd_readme`
+* :ref:`at_cmd_parser_readme`
+* :ref:`at_params_readme`
+* :ref:`modem_info_readme`
+* ``drivers/at_cmd``
+* ``drivers/lte_link_control``
+
+
+In addition, it uses the following samples:
+
+From |NCS|
+
+  * :ref:`secure_partition_manager`
+
+From nrfxlib
+
+  * :ref:`nrfxlib:bsdlib`
+
+From Zephyr
+
+
+
+References
+**********
+
+`nRF9160 Product Specification <https://infocenter.nordicsemi.com/topic/ps_nrf9160/nRF9160_html5_keyfeatures.html?cp=2_0_0>`_
+
+`nRF Connect SDK Documentation <http://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html>`_
```

Comment:
No need for this either - this is where people should be reading this. ;)

It would make sense to add a link to the AT commands reference guide though. I know this is mentioned further up, but it's quite important for this sample, so it doesn't hurt to repeat it.

Reply:
removed.